lib/db_utils.go

```diff
@@ -202,10 +202,29 @@ var (
 	_PrefixRecloutedPostHashReclouterPubKeyRecloutPostHash = []byte{46}
 	_PrefixDiamondedPostHashDiamonderPKIDDiamondLevel      = []byte{47}
 
+	// Prefixes for NFT ownership entries:
+	// 	<prefix, NFTPostHash [32]byte, SerialNumber uint64> -> NFTEntry
+	_PrefixPostHashSerialNumberToNFTEntry = []byte{48}
+	//  <prefix, PublicKey [33]byte, IsForSale bool, BidAmountNanos uint64, NFTPostHash[32]byte, SerialNumber uint64> -> NFTEntry
```

Comment:
Should this point to NFTEntry or <NFTPostHash [32]byte, SerialNumber uint64>

Reply:
Also just gut-checking: I think we're fucked into using PublicKey because posts use PublicKey right?

Reply:
Ugh... this probably means that SwapIdentity makes you lose control of your NFTs. We should probably prioritize a re-indexing of posts soon to fix this issue.

Reply:
Hmmm, I think we could actually use PKID here.  @lazynina wdyt?

Reply:
we do use PKID here -- I just didn't name it correctly. I'll fix this.

Reply:
Incredible. Thanks!